src/index.html

```diff
@@ -10,5 +10,49 @@
   </head>
```

Comment:
add font in head, because you render default font
<img width="1491" alt="Screenshot 2023-08-07 at 16 10 16" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/16553ff3-0abe-49f2-a6e6-d24614bc3a6d">

also don't forget to add font-weight 500